pkg/apis/push/v1alpha1/unifiedpushserver_types.go

```diff
@@ -13,6 +13,19 @@ type UnifiedPushServerSpec struct {
 	// Important: Run "operator-sdk generate k8s" to regenerate code after modifying this file
 	// Add custom validation using kubebuilder tags: https://book-v1.book.kubebuilder.io/beyond_basics/generating_crd.html
 
+	//When true, deployement wont be created, service with external db details will be passed to ups
+	ExternalDB bool `json:"externalDB"`
+	//Database name for external database support
+	DatabaseName string `json:"databaseName,omitempty"`
+	//Database password for external database support
+	DatabasePassword string `json:"databasePassword,omitempty"`
+	//Database user for external database support
+	DatabaseUser string `json:"databaseUser,omitempty"`
+	//Database host for external database support
+	DatabaseHost string `json:"databaseHost,omitempty"`
+	//Database port for external database support
+	DatabasePort string `json:"databasePort,omitempty"`
```

Comment:
```suggestion
	DatabasePort intstr.IntOrString `json:"databasePort,omitempty"`
```

Allowing either int or string means that users creating a CR don't need to know that they need to quote the string of digits.